yarn.lock

```diff
@@ -972,9 +972,9 @@ side-channel@^1.0.3, side-channel@^1.0.4:
     get-intrinsic "^1.0.2"
     object-inspect "^1.9.0"
 
-"soap@https://github.com/applyinnovations/node-soap.git":
+soap@applyinnovations/node-soap#1.0.0:
   version "0.43.0"
```

Comment:
is there some way we can update this with the release version?

Reply:
maybe need to add something to the package.json within the node-soap during the create release workflow